CHANGELOG.md

```diff
@@ -5,8 +5,15 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.10.1a0] - 2022-09-6
+
+
+
 ## [0.10.0] - 2022-09-2
 
+### Changed
+# Reverted min Python version to 3.9 in pyproject.toml
```

Comment:
I guess you wanted to add this to the above section. 
also better use "-" instead of "#".

Finally, if we are going to do a patch, I guess we can name the release "0.10.1"